v7 fails to compile due to Tab+SFSymbol.swift on Xcode 16.4

I tried to update to v7 in [this pull request](https://github.com/techprimate/Flinky/pull/46#issuecomment-3485487396) but it fails to compile:

```
$ xcodebuild -project Flinky.xcodeproj -scheme App -destination "platform=iOS Simulator,OS=18.6,name=iPhone 16 Pro" build | xcbeautify

...

[SFSafeSymbols] Compiling Tab+SFSymbol.swift
❌ /Users/philip/Library/Developer/Xcode/DerivedData/Flinky-fjbekzfdacmofcbvaeurhpghphar/SourcePackages/checkouts/SFSafeSymbols/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift:87:14: initializer 'init(_:systemImage:value:content:)' requires that 'LocalizedStringResource' conform to 'StringProtocol'
        self.init(titleResource, systemImage: systemSymbol.rawValue, value: value,  content: content)
             ^
❌ /Users/philip/Library/Developer/Xcode/DerivedData/Flinky-fjbekzfdacmofcbvaeurhpghphar/SourcePackages/checkouts/SFSafeSymbols/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift:117:14: initializer 'init(_:systemImage:value:role:content:)' requires that 'LocalizedStringResource' conform to 'StringProtocol'
        self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, role: role, content: content)
             ^
❌ /Users/philip/Library/Developer/Xcode/DerivedData/Flinky-fjbekzfdacmofcbvaeurhpghphar/SourcePackages/checkouts/SFSafeSymbols/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift:145:14: initializer 'init(_:systemImage:value:content:)' requires that 'LocalizedStringResource' conform to 'StringProtocol'
        self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, content: content)
             ^
❌ /Users/philip/Library/Developer/Xcode/DerivedData/Flinky-fjbekzfdacmofcbvaeurhpghphar/SourcePackages/checkouts/SFSafeSymbols/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift:175:14: initializer 'init(_:systemImage:value:role:content:)' requires that 'LocalizedStringResource' conform to 'StringProtocol'
        self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, role: role, content: content)
             ^
❌ /Users/philip/Library/Developer/Xcode/DerivedData/Flinky-fjbekzfdacmofcbvaeurhpghphar/SourcePackages/checkouts/SFSafeSymbols/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift:236:14: initializer 'init(_:systemImage:content:)' requires that 'LocalizedStringResource' conform to 'StringProtocol'
        self.init(titleResource, systemImage: systemSymbol.rawValue, content: content)
             ^
❌ /Users/philip/Library/Developer/Xcode/DerivedData/Flinky-fjbekzfdacmofcbvaeurhpghphar/SourcePackages/checkouts/SFSafeSymbols/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift:250:14: initializer 'init(_:systemImage:role:content:)' requires that 'LocalizedStringResource' conform to 'StringProtocol'
        self.init(titleResource, systemImage: systemSymbol.rawValue, role: role, content: content)
             ^

...

** BUILD FAILED **
```

On a side-note I would recommend you to set up a GitHub Actions workflow in this repository so your library doesn't fail to compile after release.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,7 +16,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Fixed
 
-- Package cannot build below Xcode's verson 26.0
+- Fix to build by pre-Xcode 26.0 versions
 
 ## [7.0.0] - 2025-10-26
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,7 +16,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Fixed
 
-- Fix to build by pre-Xcode 26.0 versions
+- Fix to build by pre-Xcode 26.0 versions (By [Phineas Guo](https://github.com/guoPhineas))
 
 ## [7.0.0] - 2025-10-26
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,7 +16,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Fixed
 
-- None
+- Package cannot build below Xcode's verson 26.0
 
 ## [7.0.0] - 2025-10-26
 

--- a/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift
+++ b/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift
@@ -84,7 +84,7 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: Value, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
-        #if canImport(FoundationModels)
+        #if compiler(>=6.2)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value,  content: content)
         #else
         self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
@@ -118,7 +118,7 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: Value, role: TabRole?, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
-        #if canImport(FoundationModels)
+        #if compiler(>=6.2)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, role: role, content: content)
         #else
         self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
@@ -150,7 +150,7 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init<T>(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: T, @ViewBuilder content: () -> Content) where Value == T?, Label == DefaultTabLabel, T : Hashable {
-        #if canImport(FoundationModels)
+        #if compiler(>=6.2)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, content: content)
         #else
         self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
@@ -184,7 +184,7 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init<T>(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: T, role: TabRole?, @ViewBuilder content: () -> Content) where Value == T?, Label == DefaultTabLabel, T : Hashable {
-        #if canImport(FoundationModels)
+        #if compiler(>=6.2)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, role: role, content: content)
         #else
         self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
@@ -249,7 +249,7 @@ public extension Tab where Value == Never, Content: View, Label: View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
-        #if canImport(FoundationModels)
+        #if compiler(>=6.2)
         self.init(titleResource, systemImage: systemSymbol.rawValue, content: content)
         #else
         self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue,  content: content)
@@ -267,7 +267,7 @@ public extension Tab where Value == Never, Content: View, Label: View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, role: TabRole?, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
-        #if canImport(FoundationModels)
+        #if compiler(>=6.2)
         self.init(titleResource, systemImage: systemSymbol.rawValue, role: role, content: content)
         #else
         self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue,  content: content)

--- a/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift
+++ b/Sources/SFSafeSymbols/Initializers/SwiftUI/Tab+SFSymbol.swift
@@ -84,7 +84,11 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: Value, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
+        #if canImport(FoundationModels)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value,  content: content)
+        #else
+        self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
+        #endif
     }
     
     /// Creates a tab that the tab view presents when the tab view's selection
@@ -114,7 +118,11 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: Value, role: TabRole?, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
+        #if canImport(FoundationModels)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, role: role, content: content)
+        #else
+        self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
+        #endif
     }
     
     /// Creates a tab that the tab view presents when the tab view's selection
@@ -142,7 +150,11 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init<T>(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: T, @ViewBuilder content: () -> Content) where Value == T?, Label == DefaultTabLabel, T : Hashable {
+        #if canImport(FoundationModels)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, content: content)
+        #else
+        self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
+        #endif
     }
         
     /// Creates a tab that the tab view presents when the tab view's selection
@@ -172,7 +184,11 @@ public extension Tab where Value: Hashable, Content: View, Label : View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init<T>(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, value: T, role: TabRole?, @ViewBuilder content: () -> Content) where Value == T?, Label == DefaultTabLabel, T : Hashable {
+        #if canImport(FoundationModels)
         self.init(titleResource, systemImage: systemSymbol.rawValue, value: value, role: role, content: content)
+        #else
+        self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue, value: value, content: content)
+        #endif
     }
 }
 
@@ -233,7 +249,11 @@ public extension Tab where Value == Never, Content: View, Label: View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
+        #if canImport(FoundationModels)
         self.init(titleResource, systemImage: systemSymbol.rawValue, content: content)
+        #else
+        self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue,  content: content)
+        #endif
     }
 
     /// Creates a tab with a `SFSymbol` and a localized string resource label.
@@ -247,7 +267,11 @@ public extension Tab where Value == Never, Content: View, Label: View {
     @available(iOS 26.0, macOS 26.0, tvOS 26.0, watchOS 26.0, *)
     @_disfavoredOverload
     nonisolated init(_ titleResource: LocalizedStringResource, systemSymbol: SFSymbol, role: TabRole?, @ViewBuilder content: () -> Content) where Label == DefaultTabLabel {
+        #if canImport(FoundationModels)
         self.init(titleResource, systemImage: systemSymbol.rawValue, role: role, content: content)
+        #else
+        self.init(String(localized: titleResource), systemImage: systemSymbol.rawValue,  content: content)
+        #endif
     }
 }
 

